test/js/bun/ffi/cc-fixture.js

```diff
@@ -19,3 +23,7 @@ const {
 if (main() !== 42) {
   throw new Error("main() !== 42");
 }
+
+if (napi_main(null) !== "Hello, Napi!") {
```

Comment:
is this going to segfault in the handle scope code?

Reply:
discussed IRL